server/src/database/scripts/sync-chefs-data.service.ts

```diff
@@ -43,6 +47,27 @@ export class SyncChefsDataService {
     return await this.formMetadataRepo.save(this.formMetadataRepo.create(data));
   }
 
+  private async createOrUpdateAttachments(data) {
+    const responseDataFileArrays = Object.values(data).filter(
+      (value) => Array.isArray(value) && value.length > 0
+    );
+    const objects = extractObjects(responseDataFileArrays, 5);
+    // TODO:
+    // Maybe there's a better way to check it
+    const files = objects.filter((obj) => 'url' in obj && 'data' in obj);
+
+    for (const file of files) {
+      const newAttachmentData = {
+        id: file.data.id,
+        url: file.url,
+        // Data's empty for now
+        data: '',
```

Comment:
Consider this example, we already have this attachment data in our database. The next day, we run the script to update the form submission, so this data is set to empty will replace the actual file data in the database which is wrong. Since data is optional, do not set the value of the data field!

Reply:
Hey Karthikk, since this has been updated by merging GRANT-79 into this, is it okay to resolve and merge into Develop? Thanks in advance!